### PR TITLE
Fix for fetcher warning at server startup 

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**

### CODE REVIEW

File: stored_requests/config/config_test.go [hunk 1]
```diff
@@ -41,12 +41,42 @@ func isMemoryCacheType(cache stored_requests.CacheJSON) bool {
...
+					},
+					PollUpdates: config.PostgresUpdatePolling{
+						Query: "test poll query",
+					},
+					FetcherQueries: config.PostgresFetcherQueries{
+						QueryTemplate: "",
+					},
+				},
+			},
+			message: "If Postgres fetcher query is not defined, but Postgres Cache init query and Postgres update polling query are defined EmptyFetcher should be returned",
```
Comment (mansinahar) [inline stored_requests/config/config_test.go:68]: Can we please also add test cases where `PostgresFetcherQueries` is supplied in the config but not `PostgresUpdatePolling` and `PostgresCacheInitializer`? And also one where all of them are supplied?

Comment (VeronikaSolovei9) [reply stored_requests/config/config_test.go]: Yes, added!

File: stored_requests/config/config_test.go [hunk 2]
```diff
@@ -41,12 +42,87 @@ func isMemoryCacheType(cache stored_requests.CacheJSON) bool {
...
+		},
 	}
-	if _, ok := fetcher.(empty_fetcher.EmptyFetcher); !ok {
-		t.Errorf("If the config is empty, and EmptyFetcher should be returned")
+
+	for _, test := range testCases {
+
+		fetcher := newFetcher(test.config, nil, &sql.DB{})
+		if fetcher == nil {
+			t.Errorf("The fetcher should be non-nil.")
```
Comment (mansinahar) [inline stored_requests/config/config_test.go:114]: Please consider using `assert.NotNil` here

File: stored_requests/config/config_test.go [hunk 3]
```diff
@@ -41,12 +42,87 @@ func isMemoryCacheType(cache stored_requests.CacheJSON) bool {
...
+		}
+		if test.emptyFetcher {
+			if _, ok := fetcher.(empty_fetcher.EmptyFetcher); !ok {
```
Comment (mansinahar) [inline stored_requests/config/config_test.go:117]: Please consider using `assert.Equals` here and in the `else` part of this condition

**Review by mansinahar — COMMENTED** [review]
LGTM in general.. just minor comments

**Review by bsardo — COMMENTED** [review]
This is looking good. Just one additional nitpicky comment to add to @mansinahar's comments.

**Review by mansinahar — APPROVED** [review]

**Review by bsardo — APPROVED** [review]

**Pull request merged**